CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All Notable changes to `php-amqplib` will be documented in this file
 
 ## [Unreleased]
 
+## 2.7.0-rc1 - 2017-05-22
+### Added
+- Increased overall test coverage.
+- Bring heartbeat support to socket connection
+- Add message delivery tag for internal ack handler
```

Comment:
It would make more sense to the users if we changed it to say "for publisher confirms".